airflow/providers/celery/provider.yaml

```diff
@@ -60,3 +60,256 @@ sensors:
 executors:
   - airflow.providers.celery.executors.celery_executor.CeleryExecutor
   - airflow.providers.celery.executors.celery_kubernetes_executor.CeleryKubernetesExecutor
+
+config:
+  celery_kubernetes_executor:
+    description: |
+      This section only applies if you are using the ``CeleryKubernetesExecutor`` in
+      ``[core]`` section above
+    options:
+      kubernetes_queue:
+        description: |
+          Define when to send a task to ``KubernetesExecutor`` when using ``CeleryKubernetesExecutor``.
+          When the queue of a task is the value of ``kubernetes_queue`` (default ``kubernetes``),
+          the task is executed via ``KubernetesExecutor``,
+          otherwise via ``CeleryExecutor``
+        version_added: 2.0.0
```

Comment:
I think this is confusing.
It's about airflow version but now that we are in providers users may think it means version 2.0.0 of Celery provider

Reply:
Right. We should remove it simply. It makes no sense to have it here.

Reply:
Removed all version_added except one which was **just** added and I marked it as version_added: 3.3.0 (upcoming celery provider version).